src/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1698,88 +1708,114 @@ msgstr "Revision"
 
-#: templates/pages/page_form.html:230
+#: templates/pages/page_form.html:236
 msgid "Parent Page"
 msgstr "Übergeordnete Seite"
 
-#: templates/pages/page_form.html:239
+#: templates/pages/page_form.html:245
 msgid "Order"
 msgstr "Reihenfolge"
 
-#: templates/pages/page_form.html:244
+#: templates/pages/page_form.html:250
 msgid "Embed live content"
 msgstr "Live-Inhalte einbinden"
 
-#: templates/pages/page_form.html:265
+#: templates/pages/page_form.html:271
 msgid "Additional permissions for this page"
 msgstr "Zusätzliche Berechtigungen für diese Seite"
 
-#: templates/pages/page_form.html:266
+#: templates/pages/page_form.html:272
 msgid ""
 "This affects only users, who don't have the permission to change arbitrary "
 "pages anyway."
 msgstr ""
 "Dies betrifft nur Benutzer, die nicht ohnehin die Berechtigung haben, "
 "beliebige Seiten zu ändern."
 
-#: templates/pages/page_form.html:283 templates/pages/page_form.html:284
-#: templates/pages/page_tree_archived_node.html:87
+#: templates/pages/page_form.html:289 templates/pages/page_form.html:290
+#: templates/pages/page_form.html:294
+#: templates/pages/page_tree_archived_node.html:92
 msgid "Restore page"
 msgstr "Seite wiederherstellen"
 
-#: templates/pages/page_form.html:285
+#: templates/pages/page_form.html:291
 msgid "Restore this page"
 msgstr "Diese Seite wiederherstellen"
 
-#: templates/pages/page_form.html:288 templates/pages/page_form.html:289
+#: templates/pages/page_form.html:297
+msgid "To restore this page, you have to restore its parent page:"
+msgid_plural ""
+"To restore this page, you have to restore all its archived parent pages:"
+msgstr[0] ""
+"Um diese Seite wiederherzustellen, müssen Sie ihre übergeordnete Seite "
+"wiederherstellen:"
+msgstr[1] ""
+"Um diese Seite wiederherzustellen, müssen Sie alle ihre archivierten "
+"übergeordneten Seiten wiederherstellen:"
+
+#: templates/pages/page_form.html:310 templates/pages/page_form.html:311
 #: templates/pages/page_tree_node.html:91
 msgid "Archive page"
 msgstr "Seite archivieren"
 
-#: templates/pages/page_form.html:291
+#: templates/pages/page_form.html:313
 msgid "Archive this page"
 msgstr "Diese Seite archivieren"
 
-#: templates/pages/page_form.html:297 templates/pages/page_form.html:301
-#: templates/pages/page_tree_archived_node.html:96
+#: templates/pages/page_form.html:319 templates/pages/page_form.html:337
+#: templates/pages/page_tree_archived_node.html:106
 #: templates/pages/page_tree_node.html:100
 msgid "Delete page"
 msgstr "Seite löschen"
 
-#: templates/pages/page_form.html:299
-#: templates/pages/page_tree_archived_node.html:92
-#: templates/pages/page_tree_node.html:96
+#: templates/pages/page_form.html:323
 msgid "You cannot delete a page which has children."
 msgstr "Sie können keine Seite löschen, die Unterseiten besitzt."
 
-#: templates/pages/page_form.html:303
+#: templates/pages/page_form.html:324
+msgid "To delete this page, you have to delete or move its subpage first:"
```

Comment:
Just stumbled upon the change from "child" to "subpage". There are still some occurences of "child" in the .po file, which we should change accordingly (lines 1804, 2925, 167, 171)?

Reply:
Good point. I had a look at WordPress and noticed that the typical wording is "subpage" instead of "child page", so probably it's a good idea to keep this consistent.